examples/puppetmaster/vagrant/README.md

```diff
@@ -20,6 +20,7 @@
   * [Install Puppet Agent on the Windows VM](#install-puppet-agent-on-the-windows-vm)
   * [Clear Previous Provisioning Results (if Necessary)](#clear-previous-provisioning-results--if-necessary-)
```

Comment:
This line (`Clear Previous Provisioning Results`) can be deleted from Table of Contents.